hack/verify-govet.sh

```diff
@@ -24,6 +24,10 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 cd "${KUBE_ROOT}"
 
+# This is required before we run govet for the results to be correct.
+# See https://github.com/golang/go/issues/16086 for details.
+go install ./cmd/...
```

Comment:
fun: we don't need to fix this in this PR, but I think building only `cmd/...` is causing us to miss some stuff:
```console
$ time go list ./... | grep -v 'third_party\|vendor' | xargs go install

real    4m2.226s
user    20m37.690s
sys     1m45.396s
$ time go list ./... | grep -v 'third_party\|vendor' | xargs go vet
federation/pkg/dnsprovider/providers/aws/route53/testing/route53api.go:79: arg change.ResourceRecordSet.Name for printf verb %s of wrong type: *string
federation/pkg/dnsprovider/providers/aws/route53/testing/route53api.go:84: arg change.ResourceRecordSet.Name for printf verb %s of wrong type: *string
federation/pkg/dnsprovider/providers/aws/route53/testing/route53api.go:105: arg input.Name for printf verb %s of wrong type: *string
exit status 1
federation/pkg/dnsprovider/providers/google/clouddns/clouddns.go:110: k8s.io/kubernetes/federation/pkg/dnsprovider/providers/google/clouddns/internal/stubs.ManagedZone composite literal uses unkeyed fields
exit status 1

real    1m39.439s
user    1m52.376s
sys     0m10.650s
```